sim/core/warrior/rotation.go

```diff
@@ -0,0 +1,15 @@
+package warrior
+
+import (
+	"time"
+
+	"github.com/wowsims/tbc/sim/core"
+)
+
+func (warrior *Warrior) OnGCDReady(sim *core.Simulation) {
+	warrior.tryUseGCD(sim)
+}
+
+func (warrior *Warrior) tryUseGCD(sim *core.Simulation) {
+	warrior.WaitUntil(sim, sim.Duration+time.Second)
+}
```

Comment:
After merging my hunter PR I realized this isn't necessary, it should be safe for you to delete this and just have an empty OnGCDReady() function.